src/css/default/_price-ranges.scss

```diff
@@ -1,3 +1,4 @@
+// sass-lint:disable no-empty-rulesets
```

Comment:
Why are they empty? Could we maybe get rid of the whole css file in some dead code elimination step?

Reply:
It's a template for those who want to use them as a starting point for making their own theme.